mobile/lib/services/hash.service.dart

```diff
@@ -65,7 +65,13 @@ class HashService {
       if (hashes[i] != null) {
         continue;
       }
-      final file = await assets[i].local!.originFile;
+
+      File? file;
+
+      try {
+        file = await assets[i].local!.originFile;
+      } catch (_) {}
```

Comment:
You should definitely log these errors to know what and why is happening

Reply:
Done